Wikipedia/Code/ViewWithFindInPageKeyboardBarInputAccessoryView.h

```diff
@@ -0,0 +1,8 @@
+#import <UIKit/UIKit.h>
+#import "FindInPageKeyboardBar.h"
+
+@interface ViewWithFindInPageKeyboardBarInputAccessoryView : UIView
```

Comment:
Ok I just looked back into this to refresh my memory... 

This approach (redeclaring `inputAccessoryView` as `readwrite`) is roughly outlined in the `inputAccessoryView` documentation: https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIResponder_Class/index.html#//apple_ref/occ/instp/UIResponder/inputAccessoryView

It lets easily transition between having the keyboardBar "docked" at the bottom of the screen or sitting atop the keyboard.